printer.py

```diff
@@ -59,6 +60,11 @@ def move(direction):
 def status(size):
     print("Your katamari is {0}cm".format(str(size)))
 
+def bump(item):
+    status=["bumps and","rubs and","wiggles then","stalls then"]
```

Comment:
not that we have linting checks but this is a linting issue (lack of spaces around a = )